Dockerfile

```diff
@@ -1,36 +1,33 @@
-FROM node:lts-alpine AS build
-RUN apk add --no-cache dumb-init
```

Comment:
hav you checked whether `dumb-init` is built idenically on `alpine` and `bullseye-slim`?

Reply:
?

Reply:
Indeed. But `dumb-init` tool is built in the `build` stage. Then on line 31, you copy the binary from `build` stage, assuming it is located at `/usr/bin/dumb-init`. Is this assumption still valid for Debian? Maybe it is, but I just wanted to point this, you may have missed it.

Reply:
it is